action/protocol/rewarding/admin.go

```diff
@@ -26,6 +32,8 @@ type admin struct {
 	foundationBonus                *big.Int
 	numDelegatesForFoundationBonus uint64
 	foundationBonusLastEpoch       uint64
+	foundationBonusP2StartEpoch    uint64
+	foundationBonusP2EndEpoch      uint64
```

Comment:
need to also put these 2 into protobuf

Reply:
also need to add these 2 into genesis, same as foundationBonusLastEpoch

Reply:
sure!